generation/diff_directory.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
```

Comment:
Add usage.

Reply:
I have already mentioned in diff_files.sh usage that diff_directory.sh is just a helper script. You do not need any instructions to run it. It will just be called by diff_files.sh and do its work, you do not need to invoke/run it yourself.
I have included these instructions in the comments for this script now.